apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -217,6 +217,35 @@ handle_request_('GetNetworkStatus', _Req, _Context) ->
             {200, [], aec_peer_analytics:get_stats_for_client()}
     end;
 
+handle_request_('GetCheckTxInPool', Req, _Context) ->
+    ParseFuns = [ parse_map_to_atom_keys(),
+                  read_required_params([hash]),
+                  api_decode([{hash, hash, tx_hash}]),
+                  fun(_Req, #{hash := TxHash}) ->
+                      case aec_chain:find_tx_with_location(TxHash) of
+                          none -> {error, {404, [], #{<<"reason">> => <<"tx_not_found">>}}};
+                          {BlockHash, _Tx} when is_binary(BlockHash) ->
+                              {ok, {200, [], #{<<"status">> => <<"included">>}}};
+                          {mempool, SignedTx} ->
+                              {Env, Trees} = aetx_env:tx_env_and_trees_from_top(aetx_transaction),
+                              Env1 = aetx_env:set_signed_tx(Env, {value, SignedTx}),
+                              Tx = aetx_sign:tx(SignedTx),
+                              try aetx:process(Tx, Trees, Env1) of
```

Comment:
What about the signature? Or is that always checked before adding to tx-pool? 🤔

Reply:
Yes, transactions with invalid signature are rejected and not allowed in the pool